src/store/reducers/tenantOverview/executeTopShards/types.ts

```diff
@@ -0,0 +1,18 @@
+import type {IQueryResult, QueryErrorResponse} from '../../../../types/store/query';
+import type {ApiRequestAction} from '../../../utils';
+import {FETCH_TOP_SHARDS, setDataWasNotLoaded} from './executeTopShards';
+
+export interface TopShardsState {
+    loading: boolean;
+    wasLoaded: boolean;
+    data?: IQueryResult;
+    error?: QueryErrorResponse;
+}
+
+export type TopShardsAction =
+    | ApiRequestAction<typeof FETCH_TOP_SHARDS, IQueryResult, QueryErrorResponse>
+    | ReturnType<typeof setDataWasNotLoaded>;
+
+export interface TopShardsSlice {
+    topShards: TopShardsState;
+}
```

Comment:
1) Your reducer is imported as `executeTopTables` to root reducer, so it's type not applicable at all
2) This type isn't used